assets/js/actions/invitation.js

```diff
@@ -1,28 +1,37 @@
-import * as rest from '../util/rest';
-import sanitizeHtml from 'sanitize-html'
+import * as rest from "../util/rest";
+import sanitizeHtml from "sanitize-html";
 
 export const inviteUser = (email, role, organization) => {
   return (dispatch) => {
     const invitation = {
       email: sanitizeHtml(email),
       role,
-      organization
-    }
-    rest.post(`/api/invitations`, { invitation })
-    .then(response => {})
-  }
-}
+      organization,
+    };
+    rest.post(`/api/invitations`, { invitation }).then((response) => {});
+  };
+};
 
 export const getInvitation = (token) => {
   return (dispatch) => {
-    return rest.get(`/api/invitations/${token}`)
-      .then(response => response.data)
-  }
-}
+    return rest
+      .get(`/api/invitations/${token}`)
+      .then((response) => response.data);
+  };
+};
 
 export const deleteInvitation = (invitation) => {
   return (dispatch) => {
-    rest.destroy(`/api/invitations/${invitation.id}`)
-      .then(response => {})
-  }
-}
+    rest.destroy(`/api/invitations/${invitation.id}`).then((response) => {});
+  };
+};
+
+export const getInvitations = async (email) => {
+  const invitations = await rest.get(`/invitations/${email}`);
+  return invitations.data;
+};
+
+export const resendInvitations = async (email) => {
+  const result = await rest.get(`/resend_invitation/${email}`);
```

Comment:
maybe this should be a post? Get seems like the wrong request type here

Reply:
ah shoot i was getting an error w/ and switched to get just to get it working and then forgot. totally agree.